src/app/ui/(authentication)/signup/page.tsx

```diff
@@ -0,0 +1,152 @@
+'use client'
+import React from 'react'
+import Link from 'next/link'
+import { Card, Input, Button, Typography } from '@material-tailwind/react'
+import { roboto } from '../../fonts'
+import { useForm } from 'react-hook-form'
+import ErrorInput from './ErrorInput'
+
+export default function page() {
+  const onSubmit = (data) => {
+    console.log(data) // Submit form data
+  }
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const {
+    register,
+    handleSubmit,
+    getValues,
+    formState: { errors },
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+  } = useForm()
+
+  return (
+    <Card
+      color="transparent"
+      shadow={false}
+      placeholder={undefined}
+      className={`${roboto} w-[20rem] m-auto pt-[10rem] `}>
```

Comment:
```suggestion
      className={`${roboto.className} w-[20rem] m-auto pt-[10rem] `}>
```
chỉnh mấy chỗ khác lun